recipe/meta.yaml

```diff
@@ -21,7 +21,7 @@ requirements:
   - bob.extension
   - bob.blitz
   - bob.core
-  - hdf5
+  - hdf5 1.8.15*
```

Comment:
Bear in mind that people installing this package in envs without `libnetcdf` or `h5py` would be OK regardless of the `hdf5` version.